src/h5web/toolbar/controls/DomainSlider/DomainSlider.test.tsx

```diff
@@ -0,0 +1,336 @@
+import { screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { pressKey, renderApp, selectExplorerNode } from '../../../test-utils';
+
+test('show slider with two thumbs', async () => {
+  renderApp();
```

Comment:
I get you but we are introducing a hidden dependency here. The domain slider tests require that all the NeXus handling works as intended. I feel that this breaks *a lot* the encapsulation of tests...

I think adding it to the first test and not to the following may be even harder to debug tests in case of a NeXus resolution failure...

Anyway, we have to make clear that the root group displays an image. So perhaps add only a comment ?

Reply:
Also changing to `nexus_entry` still relies on the NeXus parsing of the children group.

The selection should be on `/nexus_entry/nx_process/nx_data/twoD` to avoid any NeXus dependency.

I get that it is cumbersome so I don't mind if you don't put it in the end.

Reply:
No worries, since you feel strongly about it, I've added `await selectExplorerNode('nD_datasets/twoD');` to all the tests.

Reply:
I've added a comment to the first `renderApp`